CalculatorExercises/1-CalculatorBasic/Calculator.java

```diff
@@ -0,0 +1,61 @@
+public class Calculator {
```

Comment:
one more thing:
rename 'CalculatorExercises' to 'calculator' and
'1-CalculatorBasic' to 'basic'
add
package calculator.basic;
to the beginning of each file